modules/rosa-upgrading-preparing-4-8-to-4-9.adoc

```diff
@@ -1,18 +0,0 @@
-// Module included in the following assemblies:
```

Comment:
This module appears in the `upgrading/rosa-upgrading-cluster-prepare.adoc` assembly.